python/lesson3/tutorial.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title: Discovering Flask via Python and pip
+---
+
+An alternative to *Ruby's Sinatra* is the lesser known *Flask* (run using Python).
+Both are web application frameworks. Similar to the MVC (model-view-controller)
```

Comment:
This isn't really correct. You Sinatra and Flask are not inherently MVC. You could build an MVC app with them, but you could also opt to use another pattern.